oci/private/tarball.bzl

```diff
@@ -51,10 +51,9 @@ attrs = {
 
             See the _run_template attribute for the script that calls this loader tool.
             """,
-        allow_single_file = True,
         mandatory = False,
```

Comment:
allow_single_file might have been load-bearing if someone just pointed this at an executable .sh file?

Reply:
hmm, i think i am gonna revert this change.

Reply:
can be done later if needed.

Reply:
reverted.